ts/create-kpt-functions/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-kpt-functions",
-  "version": "0.16.4",
```

Comment:
we should probably also update the package.json mustache template to use the latest version of kpt-functions library as part of this (Version installed by CLI does not include structured results for example)

Let's make it 0.17.0

Reply:
Done